src/components/vertical-menu/vertical-menu-full-screen/vertical-menu-full-screen.component.tsx

```diff
@@ -90,9 +94,7 @@ export const VerticalMenuFullScreen = ({
               />
             </IconButton>
           </Box>
-          <VerticalMenuFullScreenContext.Provider
-            value={{ isFullScreen: true }}
-          >
+          <VerticalMenuFullScreenContext.Provider value={contextValue}>
```

Comment:
Ah this is no longer required - one of the new lint rules that I've since disabled wants these values to be memoised. It might be something we introduce later